### PR TITLE
Add `gulp-jsonminify` types

### PR DESCRIPTION
**Pull request by pine**
Hello. I've added [gulp-jsonminify](https://www.npmjs.com/package/gulp-jsonminify) types.

Thank you.

---

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

**Review by plantain-00 — CHANGES_REQUESTED** [review]
`tslint.json` should be present

File: types/gulp-jsonminify/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
+        "strictFunctionTypes": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "paths": {
+            "q": [
```
Comment (andy-ms) [inline types/gulp-jsonminify/tsconfig.json]: This doesn't seem to be used.

File: types/gulp-jsonminify/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```
Comment (andy-ms) [inline types/gulp-jsonminify/tsconfig.json]: Should be `true` for all new packages

**Review by plantain-00 — APPROVED** [review]

**Comment by andy-ms** [comment]
Thanks!

**Pull request merged**